job-descriptions/software-engineer.md

```diff
@@ -40,15 +40,20 @@ Take a look at our public [near-term product roadmap](https://docs.sourcegraph.c
 
 ### Interview process
 
-1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU)!
-1.  We schedule a 30 minute call to chat with you about Sourcegraph and to find out what you are looking for in your next role.
+1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU).
+1.  We set up a 30 minute call to chat with you about Sourcegraph to find out what you are looking for in your next role.
 1.  We give you a take home coding assignment which won't take you more than 2 hours to finish.
 1.  We schedule a few more hours of technical and non-technical interviews. We're happy to fly you out to our San Francisco office, or conduct the remaining interviews over video chat, whatever works best for you.
 1.  We check your references.
-1.  We make you a job offer!
+1.  We make you a job offer.

```

Comment:
Are exclamation points discouraged?

Reply:
They were– pretty heavily too (I was surprised). They can make the posting seem somewhat less professional or legitimate.